SharpSource/SharpSource.Test/AccessingTaskResultWithoutAwaitTests.cs

```diff
@@ -52,9 +48,7 @@ async Task MyMethod()
         async Task<int> Other() => 5;
     }
 }";
-
-        await VerifyDiagnostic(original, "Use await to get the result of a Task.");
-        await VerifyFix(original, result);
+        await VerifyCS.VerifyCodeFix(original, VerifyCS.Diagnostic().WithMessage("Use await to get the result of a Task."), result);
```

Comment:
In case you don't want to not verify the message, you'd do it like `await VerifyCS.VerifyCodeFix(original, result)`, and replace `{|#0:Other().Result|}` with `[|Other().Result|]`

It all depends on what you want to assert, but wanted to let you know about the shorthand.